src/app/components/team/Statuses/StatusListItem.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import IconButton from '@material-ui/core/IconButton';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
+import ListItemText from '@material-ui/core/ListItemText';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import IconMoreVert from '@material-ui/icons/MoreVert';
+import styled from 'styled-components';
+import globalStrings from '../../../globalStrings';
+import { subheading2 } from '../../../styles/js/shared';
+
+const StyledStatusLabel = styled.span`
+  color: ${props => props.color};
+  font: ${subheading2};
+  font-weight: 500;
+`;
+
+const StatusListItem = ({
+  defaultLanguage,
+  initialStatus,
+  onDelete,
+  onEdit,
+  onMakeDefault,
+  preventDelete,
+  status,
+}) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const handleClose = () => setAnchorEl(null);
+  const handleDelete = () => {
+    handleClose();
+    if (onDelete) onDelete(status);
+  };
+  const handleEdit = () => {
+    handleClose();
+    if (onEdit) onEdit(status); // TODO default props to avoid checking?
+  };
+  const handleMakeDefault = () => {
+    handleClose();
+    if (onMakeDefault) onMakeDefault(status); // TODO default props to avoid checking?
```

Comment:
Since `onDelete`, `onEdit` and `onMakeDefault` are marked as required props, do we need those `if`'s?